Editor/AGS.Editor/Components/TranslationsComponent.cs

```diff
@@ -187,6 +187,10 @@ private object UpdateTranslationsProcess(IWorkProgress progress, object translat
             List<Translation> translations = (List<Translation>)translationList;
             TranslationGenerator generator = new TranslationGenerator();
             CompileMessages errors = generator.CreateTranslationList(_agsEditor.CurrentGame);
+
+            if (errors.Count > 0) 

```

Comment:
`errors` here is CompileMessages object, which may or not contain actual errors (it may be also warnings or other notifications). Looking at the CompiledMessages class, it provides `HasErrors` property to tell if there were any errors.
Same `HasErrors` property should be checked below in `DoTranslationUpdate`.